shared/src/commonMain/kotlin/com/mbta/tid/mbta_app/model/TripDetailsStopList.kt

```diff
@@ -23,7 +23,7 @@ constructor(val tripId: String, val stops: List<Entry>, val startTerminalEntry:
         val prediction: Prediction?,
         // The prediction stop can be the same as `stop`, but it can also be a child stop which
         // contains more specific boarding information for a prediction, like the track number
-        val predictionStop: Stop?,
+        val predictionStop: Stop? = stop.takeIf { prediction != null },
```

Comment:
A prior draft involved creating a new `scheduleStop`, which would be completely unwieldy with no default, so I gave both `scheduleStop` and `predictionStop` a default, and then I noticed that the `Entry.stop` is not actually the parent station and in fact will always be the schedule stop if the prediction stop is different, so I removed `scheduleStop`, but I think giving the prediction stop a default value makes the tests simpler, so I’m leaving that in.